packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -125,12 +117,7 @@ class GoogleSignIn {
   /// The [hostedDomain] argument specifies a hosted domain restriction. By
   /// setting this, sign in will be restricted to accounts of the user in the
   /// specified domain. By default, the list of accounts will not be restricted.
-  ///
-  /// The [clientId] argument specifies the ID that was assigned to your app
-  /// when it was registered with Google. It is optional; if unspecified, it
-  /// will be deduced based on your project's metadata (e.g.
-  /// `GoogleServices-Info.plist` on iOS).
-  GoogleSignIn({this.scopes, this.hostedDomain, this.clientId});
+  GoogleSignIn({this.scopes, this.hostedDomain});
```

Comment:
Is there a deprecation path for external users?

We should probably at least put a comment here that ClientId has been deprecated and refer to readme for instructions on how to create the config file.